### PR TITLE
Allow CEO dropdowns to change queries

### LINKED ISSUE #4207: when you type in the text editor, it disconnects from the builder at the bottom
as a user of the data explorer / cell editor
with editor permissions or higher
when I type in the text editor for a query
then i would like it to not disconnect from the builder at the bottom of the page

### PR DESCRIPTION
**Pull request by bthesorceror**
Closes #4207

  - [x] Rebased/mergeable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by alexpaxton** [comment]
Can you post a screenshot or gif of what this does?

**Comment by kyawthura18115** [comment]
Automobileengineeringservices@googlemail.com

File: ui/src/shared/components/FillQuery.tsx [hunk 1]
```diff
@@ -37,6 +38,28 @@ class FillQuery extends PureComponent<Props, State> {
...
+      return {
+        selected: queryFills.find(fill => fill.type === NUMBER),
+        currentNumberValue: props.value,
+        resetNumberValue: props.value,
+        value: props.value,
+      }
+    }
+
+    return {
+      selected: queryFills.find(fill => fill.type === props.value),
```
Comment (ischolten) [inline ui/src/shared/components/FillQuery.tsx]: could the function used to find selected ever not find something?

File: ui/src/shared/components/TimeMachine/TimeMachine.tsx [hunk 2]
```diff
@@ -483,7 +483,7 @@ class TimeMachine extends PureComponent<Props, State> {
     return isInCEO ? QueryUpdateState.CEO : QueryUpdateState.DE
   }
 
-  private getActiveQuery = (): QueryConfig => {
+  private get activeQuery(): QueryConfig {
```
Comment (ischolten) [inline ui/src/shared/components/TimeMachine/TimeMachine.tsx]: nice!

File: ui/src/shared/components/FillQuery.tsx (hunk 1 above)
Comment (bthesorceror) [reply ui/src/shared/components/FillQuery.tsx]: yes but a null value should just mean that the first item in the list is selected

**Pull request merged**